default/scripting/monster_designs/ShipDesignManifest.focs.txt

```diff
@@ -0,0 +1,36 @@
+ShipDesignManifest
```

Comment:
It should not be stored in the resources directory then. It belongs in the user data directory.